diff -up a/common-project-config.pri.cxxflags b/common-project-config.pri
--- a/common-project-config.pri.cxxflags	2018-09-17 11:28:27.000000000 -0500
+++ b/common-project-config.pri	2020-11-08 12:40:44.453972891 -0600
@@ -12,7 +12,7 @@ RCC_DIR         = resources
 #UI_SOURCES_DIR  = ui/src
 
 # we don't like warnings...
-QMAKE_CXXFLAGS -= -Werror -Wno-write-strings
+#QMAKE_CXXFLAGS -= -Werror -Wno-write-strings
 # Disable exceptions
 CONFIG += exceptions_off
 # Use C++11
